### PR TITLE
Add deprecated message for micronaut generator and links to actual micronaut generator

### PR DESCRIPTION
**Pull request by altro3**

### CODE REVIEW

**Comment by altro3** [comment]
@wing328 look to this, please

**Review by scprek — APPROVED** [review]
Should the website also be updated to put a deprecated notes on the generators list page and on the details page (unless that is derived from the Java docs?).

**Comment by altro3** [comment]
@scprek done

**Review by scprek — APPROVED** [review]

**Comment by scprek** [comment]
Excellent. This will help a lot!

**Comment by wing328** [comment]
Have you talked to @andriy-dmytruk (creator of the generator) about this change?

I understood there are "official" implementation but there may be users out there preferring to use this one instead.

**Comment by altro3** [comment]
@wing328 It was he who moved it to micronaut. Because he works at Oracle, and micronaut is now an Oracle product.

It's the same generator, it's just that this openapi generator is now updated very rarely and we couldn't wait so long for releases to fix bugs. So we decided to move it to our repository so that we could quickly fix and improve the generator.

I'm just continuing the work of @andriy-dmytruk . He's working on other projects now.

In other words: yes, he knows :-)

**Comment by wing328** [comment]
thanks for the details. we will follow the typical procedure to deprecate the generator.

have a nice weekend

**Comment by altro3** [comment]
@wing328 could you help? What is wrong here: https://github.com/OpenAPITools/openapi-generator/actions/runs/11855289464/job/33079367106

**Comment by andriy-dmytruk** [comment]
Yes, I confirm this looks good. Historically, we moved the generator to Micronaut repo when Micronaut 3.x was released in order to make it simpler to make changes to it, control what is merged and integrate with micronaut maven and gradle plugins better.

The generator in this repo is for Micronaut 2.x and it isn't really maintained, so it makes sense to deprecate it to me.

Thanks @altro3

**Comment by wing328** [comment]
OK. I'll work on this PR and get it merged by the end of this week.
